x/ecocredit/server/core/mint_batch_credits.go

```diff
@@ -25,12 +26,18 @@ func (k Keeper) MintBatchCredits(ctx context.Context, req *core.MsgMintBatchCred
 	}
 
 	if err := k.assertCanMintBatch(issuer, batch); err != nil {
-		return nil, sdkerrors.ErrInvalidRequest.Wrapf("unable to mint credits: %s", err.Error())
+		return nil, fmt.Errorf("unable to mint credits: %s", err.Error())
```

Comment:
`assertCanMintBatch` already returns `ErrInvalidRequest` or `ErrUnauthorized`

Reply:
returning `fmt.Errorf` in message server returns `undefined` codespace, similar to #1215.

Reply:
Hmmm... using `ErrInvalidRequest` was retureing `...: invalid request: invalid request`. Will take another look but I think we should avoid stacking the same error message.

See [this line](https://github.com/regen-network/regen-ledger/blob/5eb57d5cc27e6489d051bb26d3a5930cf5389bf0/x/ecocredit/server/core/features/msg_mint_batch_credits.feature#L18), which is the expected error with this change.

Reply:
I also think #1215 has to do with an invariant violation, most likely resetting escrowed, but will have another look.

Reply:
Looks like we can use `sdkerrors.Wrap`, which will inherit the codespace of the wrapped error.

```
Wrap extends given error with an additional information.
If the wrapped error does not provide ABCICode method (ie. stdlib errors), it will be labeled as internal error.
```

Updated in https://github.com/regen-network/regen-ledger/pull/1229/commits/8dfd2912f4629a05ba588d5c2589a04b4e06c49d